lib/tasks/email_checkout_reminder.rake

```diff
@@ -0,0 +1,19 @@
+desc 'Send email reminder about upcoming checkout'
```

Comment:
yup. it would be easy to refactor all the email tasks into the same file and testable though